Loader/SymfonyFixturesLoader.php

```diff
@@ -66,11 +66,9 @@ public function addFixture(FixtureInterface $fixture): void
     /**
      * Overridden to not allow new fixture classes to be instantiated.
      *
-     * @param string $class
-     *
-     * @return FixtureInterface
+     * @phpcsSuppress SlevomatCodingStandard.TypeHints.ParameterTypeHint.MissingAnyTypeHint
```

Comment:
I hope will be be fixable with https://github.com/doctrine/data-fixtures/pull/455